polygon/sync/sync.go

```diff
@@ -123,23 +123,19 @@ func (s *Sync) commitExecution(ctx context.Context, newTip *types.Header, finali
 	return nil
 }
 
-func (s *Sync) handleMilestoneTipMismatch(
-	ctx context.Context,
-	ccBuilder CanonicalChainBuilder,
-	milestone EventNewMilestone,
-) error {
+func (s *Sync) handleMilestoneTipMismatch(ctx context.Context, ccb *CanonicalChainBuilder, ms EventNewMilestone) error {
```

Comment:
The old names were more descriptive.

Reply:
The function names got very long and spread on many lines unnecessary for just 3 input args so wanted to make them a bit more concise. I think `ccb` is a nice abbreviation (like `ufc` for update fork choice) once you understand the context.  Agree `ms` is not so clear, so I renamed that to `event` for consistency with the other functions